src/diffusers/pipelines/stable_diffusion_3/pipeline_stable_diffusion_3.py

```diff
@@ -323,6 +324,7 @@ def encode_prompt(
         pooled_prompt_embeds: Optional[torch.FloatTensor] = None,
         negative_pooled_prompt_embeds: Optional[torch.FloatTensor] = None,
         clip_skip: Optional[int] = None,
+        max_sequence_length: Optional[int] = None,
```

Comment:
Why set this optional?

Reply:
The same as the other answer I wrote, if the users don't pass it, the `_get_t5_prompt_embeds` function will set it to 256. I only did it in the case we need to change the value in the future, we only have to do it in one place and not all over the file. Since this is related to the other question, if both @yiyixuxu and you agree, I can set it to 256 in all the places where it's used.

Reply:
I am okay with that especially since it will almost always positively impact the results. So, will defer to @yiyixuxu for the final call.

Reply:
oh to not delay it more, I just applied all the suggestions, so now all of them have the 256 set and aren't optional. It is not that much of a change at the end, so what works best with the current codebase is fine for me.